float32 hx_output
float32 gamma_output
float32 v_output
float32 w_output
